docs/DiscreteMathematics/set-theory-sets.md

```diff
@@ -0,0 +1,128 @@
+# 集合论——集合
+
+不要被**集合论**的大标题吓到，我们仅仅涉及集合论一点点知识，本章所介绍的知识主要用来研究离散结构、表示离散对象。至于名字，你可以叫“集合论”，也可以叫“关系代数”，或者你也可以把他们拆散成单独的小章节去理解。许多重要的离散结构都建立在**集合**——汇集对象的结构的基础上，而**集合论**也是笔者称呼这些章节的习惯。在此之上，你还会认识到**组合**——广泛用于无序元素计数、**关系**——序偶的集合用来表示对象的关系、**有限状态机**——为计算机建模。
+
+## 集合
+
+**集合**：集合是对象的一个无序的聚集。更严谨的英文定义如下:
+
+!!! note "Defination of set"
```

Comment:
> __[Misspell]__ "Defination" is a misspelling of "Definition" ([view](https://sider.review/gh/repos/252671407/pulls/25#issue-4625798))
>
> <details>
>
> | Rule | Correct | Incorrect |
> | ---- | ------- | --------- |
> | `Definition` | Definition | Defination |
>
> You can [close this issue](https://sider.review/gh/repos/252671407/pulls/25#issue-4625798) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

It will be fixed in the next commit.